components/PrimarySourceSetsComponents/PSSFooter/index.js

```diff
@@ -21,7 +21,14 @@ const OtherInfo = () =>
           <a className="link">
             DPLA's Education Advisory Committee.
           </a>
-        </Link>
+        </Link>{" "}
+        <span>
+          Explore resources and ideas for
+          <Link href="/about/using-dplas-primary-source-sets">
+            <a> Using DPLA's Primary Source Sets</a>
```

Comment:
should that be a lowercase "u" in "using"?

Reply:
also, will space before "using" be underlined as part of the link if its included in the <a> tag?

Reply:
I would think so. @samanthadpla?

Reply:
Keep U uppercase because it's the title of the linked page. Do not link the space before the U.